### PR TITLE
murmur3: add benchmarks

### PR DESCRIPTION
**Pull request by ripatel-fd**

### CODE REVIEW

File: src/ballet/murmur3/test_murmur3.c [hunk 1]
```diff
@@ -62,6 +64,51 @@ main( int     argc,
                    "\n\t\t", sz, hash, expected ));
   }
 
+  FD_LOG_NOTICE(( "Benchmarking small inputs" ));
+
+  /* warmup */
+  uint hash = 42;
```
Comment (ptaffet-jump) [inline src/ballet/murmur3/test_murmur3.c:70]:
```suggestion
  uint hash = 42U;
```

File: src/ballet/murmur3/test_murmur3.c [hunk 2]
```diff
@@ -62,6 +64,51 @@ main( int     argc,
...
+  for( ulong i=0; i<100000UL; i++ ) {
```
Comment (ptaffet-jump) [inline src/ballet/murmur3/test_murmur3.c:71]:
```suggestion
  for( ulong i=0UL; i<100000UL; i++ ) {
```

File: src/ballet/murmur3/test_murmur3.c [hunk 3]
```diff
@@ -62,6 +64,51 @@ main( int     argc,
...
+    uint x[2] = { (uint)i, hash };
+    hash = fd_murmur3_32( &x, 8UL, 0 );
+  }
+
+  /* for real */
+  ulong bench_cnt = 100000000UL;
+  long dt = -fd_log_wallclock();
+  for( ulong i=0; i<bench_cnt; i++ ) {
```
Comment (ptaffet-jump) [inline src/ballet/murmur3/test_murmur3.c:79]:
```suggestion
  for( ulong i=0UL; i<bench_cnt; i++ ) {
```

File: src/ballet/murmur3/test_murmur3.c [hunk 4]
```diff
@@ -62,6 +64,51 @@ main( int     argc,
...
+  }
+  FD_COMPILER_FORGET( hash );
+  dt += fd_log_wallclock();
+  FD_LOG_NOTICE(( "%.3f ns/hash (sz 8)", (double)((float)dt / (float)bench_cnt) ));
+
+  FD_LOG_NOTICE(( "Benchmarking hashrate" ));
+
+  uchar msg[ 1024UL ];
+  ulong sz = 1024UL;
+  for( ulong i=0; i<sz; i+=8 )
```
Comment (ptaffet-jump) [inline src/ballet/murmur3/test_murmur3.c:91]:
```suggestion
  for( ulong i=0UL; i<sz; i+=8UL )
```

**Review by ptaffet-jump — COMMENTED** [review]
I didn't run it, but it seems good

**Review by ptaffet-jump — APPROVED** [review]
Sorry, meant to approve with the suggested optional changes, but I accidentally clicked comment.

**Review by jumpsiegel — APPROVED** [review]

**Pull request merged**